### ISSUE TITLE
TODO: Mini-tutorial on VS Code

### ISSUE THREAD

**Issue by DylanLacey** [espanso/website#41]
Given VS Code's popularity with developers, and misbehaviour when not used with the clipboard `backend` (https://github.com/espanso/espanso/issues/675) we could add a wee tutorial on writing a VS Code specific config file.

It'd give us a few advantages:

1. Easy to point people to when they're having VS Code problems
1. A nice way to demonstrate app-specific config exists and is easy
1. Could start a "Cookbook" style section
1. Something to share in dev circles to attract more assistance :P

**Comment by AucaCoyan** [comment]
Yes, I agree we can give some advice on the most used apps. Especially in those with lots of typing, like popular chat apps, editors and terminals.
As a milestone, what do you think on _first_ trying some simple text replace on every app, and then jump off to replacing html, markdown, final cursor position and so on and so forth?

**Comment by smeech** [comment]
There was a poor response to my requests on Reddit and Discord for users' app-specific configuration experiences.  

New users are likely to start with simple text editors or [EspansoEdit](https://ee.qqv.com.au/), and experienced VS Code and VSCodium users (myself now included in the latter!) aren't likely to need the assistance.

The best Cookbook currently is [EspansoEdit's](https://ee.qqv.com.au/usage/cookbook/) and a there is a selection of more esoteric scripts in my [PasteBin](https://pastebin.com/u/smeech).

For anyone coming across this, I find the following works for me:
```yml
filter_class: VSCodium
backend: clipboard
```
If I knew this were widely the case, I could include a note in the [docs](https://espanso.org/docs/tools/).

I'm inclined to close this issue, but am happy to reopen if needed.
